src/components/DateTimePicker/Internal/Tests/util/commonUtil.tsx

```diff
@@ -147,22 +148,30 @@ export function closePicker(container: HTMLElement, index = 0) {
 }
 
 export function isOpen() {
-  const dropdown = document.querySelector('.trigger-popup');
-  return dropdown && !dropdown.classList.contains('trigger-popup-hidden');
+  if (canUseDocElement()) {
+    const dropdown = document.querySelector('.trigger-popup');
+    return dropdown && !dropdown.classList.contains('trigger-popup-hidden');
+  }
+  return false;
 }
 
 export function findCell(text: string | number, index = 0) {
   let matchCell: HTMLElement;
 
-  const table = document.querySelectorAll('table')[index];
-
-  Array.from(table.querySelectorAll('td')).forEach((td) => {
-    if (td.textContent === String(text) && td.className.includes('-in-view')) {
-      matchCell = td;
+  if (canUseDocElement()) {
```

Comment:
will these functions actually get called before a document exists? or just being proactive and wrapping every references to document and window?

Reply:
Being proactive.

Reply:
Can revert, part of UTs